src/autorest-core/lib/configuration.ts

```diff
@@ -309,18 +310,32 @@ export class ConfigurationView {
       // update source locations to point to loaded Swagger
       if (m.Source) {
         const blameSources = m.Source.map(s => {
-          try {
-            const blameTree = this.DataStore.Blame(s.document, s.Position);
-            const result = [...blameTree.BlameInputs()];
-            if (result.length > 0) {
-              return result.map(r => <SourceLocation>{ document: r.source, Position: Object.assign(TryDecodeEnhancedPositionFromName(r.name) || {}, { line: r.line, column: r.column }) });
+          let posClone = Clone(s.Position);
+          while (true) {
+            try {
+              const blameTree = this.DataStore.Blame(s.document, posClone);
+              const result = [...blameTree.BlameInputs()];
```

Comment:
hmmm I'm worried that this and the lines after are inside of the `try`! bugs in those lines would be swallowed (note that you also don't write the caught exception anywhere)